lib/util/xml_utils.js

```diff
@@ -174,10 +174,11 @@ shaka.util.XmlUtils = class {
   /**
    * Parses an XML date string.
    * @param {string} dateString
+   * @param {boolean} subSecond Whether sub-second timing is allowed.
```

Comment:
Why would this ever be false?  If there's a millisecond-precise string, why chop off the milliseconds?

Reply:
Done.